### PR TITLE
WIP: set tmpfs volume rootcontext correctly

### PR DESCRIPTION
**Pull request by pmorie**
For #6238

### CODE REVIEW

**Comment by pmorie** [comment]
@thockin @markturansky @pweil- @erictune Looking for feedback on this factoring; still need to test a couple permutations, add unit tests, and add e2e for tmpfs test.

File: pkg/volume/empty_dir/empty_dir.go [hunk 1]
```diff
@@ -195,7 +202,33 @@ func (ed *emptyDir) setupTmpfs(dir string) error {
...
+	glog.V(3).Infof("pod %v: mounting tmpfs with opts %v", ed.podUID, opts)
+	return ed.mounter.Mount("tmpfs", dir, "tmpfs", 0, opts)
+}
+
+func (ed *emptyDir) getTmpfsMountOptions() (string, error) {
+	if !selinux.SelinuxEnabled() {
+		return "", nil
+	}
+
+	rootContext, err := selinux.Getfilecon(ed.rootDir)
```
Comment (thockin) [inline pkg/volume/empty_dir/empty_dir.go]: Comment on what this is doing?

**Comment by thockin** [comment]
LGTM overall

File: pkg/volume/empty_dir/empty_dir.go (hunk 1 above)
Comment (pmorie) [inline pkg/volume/empty_dir/empty_dir.go]: @thockin You mean this code isn't self-documenting? :dizzy_face:

**Comment by pweil-** [comment]
LGTM

**Comment by pmorie** [comment]
Pulled out my initial stab at the e2e tests into #6362

**Comment by yujuhong** [comment]
Could you rebase? Thanks!

**Comment by pmorie** [comment]
@yujuhong I'm pretty sure that I will close this out, actually.  I'm waiting for feedback on #6379 which I expect to obviate this.  I will close or rebase in the next 6 hours or so.

**Comment by rjnagal** [comment]
@pmorie shall we close this now?

**Comment by pmorie** [comment]
@rjnagal Yep

**Pull request closed without merging**